downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md

```diff
@@ -50,16 +50,16 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.2)](
 
 ### Improvements
 
-Introduced the WSDL CLI-tool as an experimental feature which generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from Ballerina Central and running it with an input WSDL file.
+Introduced the WSDL CLI tool as an experimental feature which generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from Ballerina Central and running it with an input WSDL file.
```

Comment:
It seems we have added this new tool under the `improvements` section. Not sure if this should come under `New Features` instead. 

IIRC, sometimes back we agreed thats its okay to ship tooling features in patch releases. So mentioning it under new features should be fine IMO. @anupama-pathirage @keizer619 wdyt?

Reply:
+1 to move under new feature,

Reply:
+1 to move it to new feature